tools/releasing/packaging/docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Dockerfile to build a ubuntu image containing the installed Debian package of a release 
-ARG branch=develop
-ARG from=precice/precice:latest
+ARG PRECICE_TAG=develop
+ARG from=precice/precice:${PRECICE_TAG}
```

Comment:
It looks like currently the CI is always using the `latest` tag of the preCICE Docker image, which is currently (correctly) v2.5.0.

This is an update we anyway should have, but it may already trigger some error in the CI.

Reply:
I mean i already pushed a `precice/python-bindings:latest` image yesterday and for me the fenics adapter built nicely with it.